### PR TITLE
Added estimated time to win to stats page

### PR DESCRIPTION
**Pull request by feydkin**
Implemented feature request @ ticket #16
Replaced block height box with an estimated time to win display ported directly from Chia. Calculation uses a static block timing - doesn't use minutes_to_win. Other TODO items are NOT completed at this time.

### CODE REVIEW

File: src/app/services/api/info.service.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { PoolInfo } from '../../interfaces/pool-info.interface';
+
+@Injectable({ providedIn: 'root' })
+export class InfoService {
+  constructor(private http: HttpClient) {}
+
+  getInfo(): Observable<PoolInfo> {
+    return this.http.get<PoolInfo>(`${environment.apiRoot}/info`);
+  }
+}
```
Comment (undsoft) [inline src/app/services/api/info.service.ts:14]: I think you can nuke StatsService now.

File: src/app/pages/stats-page/stats-page.component.ts [hunk 2]
```diff
@@ -17,23 +17,64 @@ import { StatsStore } from './stats.store';
...
+      let hours = Math.floor(minutes / hour_minutes);
+  
+      if (years > 0) { return StatsPageComponent.format_unit(minutes, "year", years, year_minutes, "month", months_minutes); }
+      if (months > 0) { return StatsPageComponent.format_unit(minutes, "month", months, months_minutes, "week", week_minutes); }
+      if (weeks > 0) { return StatsPageComponent.format_unit(minutes, "week", weeks, week_minutes, "day", day_minutes); }
+      if (days > 0) { return StatsPageComponent.format_unit(minutes, "day", days, day_minutes, "hour", hour_minutes); }
+      if (hours > 0) { return StatsPageComponent.format_unit(minutes, "hour", hours, hour_minutes, "minute", 1); }
+      if (minutes > 0) { return StatsPageComponent.format_unit_string("minute", minutes); }
+  
+      return "Unknown";
```
Comment (undsoft) [inline src/app/pages/stats-page/stats-page.component.ts:78]: 1. The angular way would be to make this reusable by making this a pipe:
See xch.pipe.ts or https://angular.io/guide/pipes
2. We also have date-fns, and it has https://date-fns.org/v2.22.1/docs/formatDuration, which may or may not be helpful.
3. I know we don't have a linter yet, but would be cool to have variables and method names in camelCase, similar to other codebase.

**Review by undsoft — CHANGES_REQUESTED** [review]
Thanks for the contribution!
Added some reorg comments.

File: src/app/pages/stats-page/stats-page.component.ts (hunk 2 above)
Comment (feydkin) [reply src/app/pages/stats-page/stats-page.component.ts]: 1. Done
2. Going to have to sit down and read through the docs on this one, still using my port from Chia at this time
3. I fixed the method names but left the internal variables for now, will beautify as time allows

**Review by undsoft — APPROVED** [review]
Looks good! Maybe let's change "Now" to "Any time now" when minutes is 0, otherwise it's kinda weird.
You can merge when ready and it'll deploy.

**Comment by william-gr** [comment]
Thank you very much!

**Pull request merged**